dts/common/nordic/nrf54h20.dtsi

```diff
@@ -429,6 +486,34 @@
 				global-domain-id = <12>;
 			};
 
+			cpuflpr_vpr: vpr@8d4000 {
+				compatible = "nordic,nrf-vpr-coprocessor";
+				reg = <0x8d4000 0x1000>;
+				status = "disabled";
+				cpu = <14>;
+				#address-cells = <1>;
+				#size-cells = <1>;
+				ranges = <0x0 0x8d4000 0x4000>;
+
+				cpuflpr_vevif_tx: mailbox@0 {
+					compatible = "nordic,nrf-vevif-task-tx";
+					reg = <0x0 0x1000>;
+					status = "disabled";
+					#mbox-cells = <1>;
+					nordic,tasks = <32>;
+					nordic,tasks-mask = <0xffff0000>;
+				};
+
+				cpuflpr_clic: interrupt-controller@1000 {
```

Comment:
This value is still incorrect.

Reply:
@57300 we need to solve this, CLICs need app level visibility due to e.g. https://github.com/zephyrproject-rtos/zephyr/pull/75451/commits/8b1be5057897bb33b52cc3dee597ab7813884201, so that regtool can route IRQs to the right core, if all have the same address, how will things work?

Reply:
`@0` is not address-less :-)

Reply:
Right :-)
I meant no bus address meaning no `reg` property. Similar to `cpu` nodes. We could do `cpufpr_clic: interrupt-controller {` as well. I think it would be compliant with dt spec. But let's start by figuring out if CLIC is available from other CPUs or not. Waiting for @57300 's answer in similar question here: https://github.com/zephyrproject-rtos/zephyr/pull/76641#discussion_r1703841262

Reply:
The value was incorrect for PPR as well. I proposed a solution in #76691.

Reply:
Implemented @57300 solution here as well.

Reply:
so.... should I go back to initial impl? This means we can't check FLPR address against MDK.

Reply:
Ok, wait a sec. Now I'm cross-checking and I was mistaken. `@1000` is correct for nRF9230. For nRF54H20 0xF0000000 is the correct address, meaning this is unavailable from cpuapp. So @57300 seems fine